<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BLE Communication GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Setup</string>
    </property>
    <widget class="QLabel" name="lblConStats">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisconnect">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBles">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection Status :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefresh">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Direction Indicator</string>
    </property>
    <widget class="QLabel" name="img_left">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/triangle_gray_left.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_right">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/triangle_gray_right.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_up">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/triangle_gray_up.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_down">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/triangle_gray_down.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
